src/package/packageVersion.ts

```diff
@@ -242,8 +244,12 @@ export class PackageVersion {
     try {
       return await pollingClient.subscribe<PackageVersionCreateRequestResult>();
     } catch (err) {
+      const report = await this.getCreateStatus(createPackageVersionRequestId, connection);
```

Comment:
Why make a call we just made when it errors?  Why not just save the result from the first call?  I would move it back the way it was.

Reply:
@shetzel This is to get the most recent, given the poll timeout or something else. And it is only done on the catch.